R/tm_a_pca.R

```diff
@@ -105,7 +105,7 @@ tm_a_pca <- function(label = "Principal Component Analysis",
                      size = c(2, 1, 8),
                      pre_output = NULL,
                      post_output = NULL) {
-  logger::log_info("Initializing tm_a_pca")
+  message("Initializing tm_a_pca")
```

Comment:
![image](https://github.com/insightsengineering/teal.logger/assets/6959016/fcb107ef-e78f-48f0-aaa9-3fcc89f4c850)

`messages` are logged as is. Looks like changes in `teal.logger` do not have any effect

Reply:
this was fixed in teal.logger